src/Console/ConfigurationResolver.php

```diff
@@ -784,4 +784,36 @@ private function setOption($name, $value)
 
         $this->options[$name] = $value;
     }
+
+    /**
+     * @param string $optionName
+     *
+     * @return bool
+     */
+    private function resolveOptionBooleanValue($optionName)
+    {
+        $value = $this->options[$optionName];
+        if (is_bool($value)) {
+            return $value;
```

Comment:
here is one example;
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.2/tests/Console/ConfigurationResolverTest.php#L1006

from the CLI we get `null` or a `string` but never booleans AFAICS